src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -328,6 +328,26 @@
             -n MyHttpSettings --probe MyNewProbe
 """
 
+helps['network application-gateway identity'] = """
+type: group
+short-summary: Manage the identity information
```

Comment:
Do you need a long summary like vm identity ‘This is required to authenticate and interact with other Azure services using bearer tokens.'?

Reply:
I need confirm with service team. For know, the identity is used to access the key vault.